1228-public_page/1242-request_draft_doi_button_is_disabled_for_publication_with_existing_doi.feature

```diff
@@ -1,11 +1,10 @@
-Feature: Request/Draft Doi buttion is disabled for Publications with existing DOI
+Feature: Request/Draft DOI button is disabled for Publications with existing DOI
 
     @1242
-    Scenario: Request/Draft Doi buttion is disabled for Publications with existing DOI
-        Given that the Creator navigates to the Public Page for Publication
+    Scenario: Request/Draft DOI button is disabled for Publications with existing DOI

```

Comment:
Request DOI and Request Draft DOI seem like different scenarios? The difference being one is PUBLISHED?

Reply:
Yes, but his is basically the same thing for the user. It will appear as the same action, but the button should have different text